changelog.d/loki-sink-default-out-of-order-action-accept.md

```diff
@@ -0,0 +1,5 @@
+The loki sink's default out_of_order_action option has been changed from deny to
```

Comment:
This filename should be `changelog.d/loki-sink-default-out-of-order-action-accept.breaking.md`. See https://github.com/vectordotdev/vector/blob/master/changelog.d/README.md